doc/tutorials/gpu-passthru.rst

```diff
@@ -0,0 +1,147 @@
+.. _gpu-passthrough:
+
+Enable GVT-d in ACRN
+#########################################
+
+This tutorial describes how to enable GVT-d in ACRN.
+
+Introduction
+*****************
+Intel GVT-d is one flavor of graphics virtualization approaches also
+names as Intel-Graphics-Device pass-through feature.
+It based on Intel VT-d technology,
+and some special graphics related configuration.
+This flavor allows direct assignment of an entire GPU’s prowess to a single user VM,
```

Comment:
When GPU passed-through to SOS, is it also called GVT-d?

Reply:
others LGTM.

Reply:
I think when  GPU pass-through to SOS, it isn't GVT-d. 
This definition comes from: https://github.com/intel/gvt-linux/wiki/GVTd_Setup_Guide#1-introduction